lib/Model/ProjectContributor.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Model;
+
+use Doctrine\SkeletonMapper\Hydrator\HydratableInterface;
+use Doctrine\SkeletonMapper\Mapping\ClassMetadataInterface;
+use Doctrine\SkeletonMapper\Mapping\LoadMetadataInterface;
+use Doctrine\SkeletonMapper\ObjectManagerInterface;
+
+class ProjectContributor implements HydratableInterface, LoadMetadataInterface
+{
+    /** @var null|TeamMember */
+    private $teamMember;
+
+    /** @var string */
+    private $projectSlug;
+
+    /** @var Project */
+    private $project;
+
+    /** @var string */
+    private $github;
+
+    /** @var string */
+    private $avatarUrl;
+
+    /** @var int */
+    private $numCommits;
+
+    /** @var int */
+    private $numAdditions;
+
+    /** @var int */
+    private $numDeletions;
+
+    public static function loadMetadata(ClassMetadataInterface $metadata) : void
+    {
+        $metadata->setIdentifier(['projectSlug', 'github']);
+    }
+
+    /**
+     * @param mixed[] $projectContributor
+     */
+    public function hydrate(array $projectContributor, ObjectManagerInterface $objectManager) : void
```

Comment:
Types for the properties should be handled in this method or we can get an invalid object state.